README.md

```diff
@@ -7,10 +7,11 @@ reality.
 ## Running locally
 
 Once you have a clone of this repository locally, navigate to the folder the project was cloned
-to and run `bin/www`. Like so:
+to and run the commands:
 
 ```
-$ bin/www
+$ npm install
+$ npm start
```

Comment:
Oh, yes, good idea. Relying on npm to provide support for starting the app is good, since `www` will probably continue to change over time.